src/v2/api/index.md

```diff
@@ -1833,104 +1833,104 @@ type: api
+  Actualiza el `textContent` del elemento. Si usted necesita actualizar parte del `textContent`, usted debe utilizar interpolaciones `{% raw %}{{ Mustache }}{$ endraw %}`.
 
-- **Example:**
+- **Ejemplo:**
 
   ```html
   <span v-text="msg"></span>
-  <!-- same as -->
+  <!-- equivale a -->
   <span>{{msg}}</span>
   ```
 
-- **See also:** [Data Binding Syntax - Interpolations](../guide/syntax.html#Text)
+- **Vea también:** [Sintaxis de Template - Interpolaciones](../guide/syntax.html#Texto)
 
 ### v-html
 
-- **Expects:** `string`
+- **Espera:** `string`
 
-- **Details:**
+- **Detalles:**
 
-  Updates the element's `innerHTML`. **Note that the contents are inserted as plain HTML - they will not be compiled as Vue templates**. If you find yourself trying to compose templates using `v-html`, try to rethink the solution by using components instead.
+  Actualiza el `innerHTML` del elemento. **Note que los contenidos son insertados como HTML puro - no serán compilados como pantillas (_templates_) de Vue**. Si usted se encuentra intentando componer plantillas utilizando `v-html`, intente pensar una solución utilizando componentes.
 
-  <p class="tip">Dynamically rendering arbitrary HTML on your website can be very dangerous because it can easily lead to [XSS attacks](https://en.wikipedia.org/wiki/Cross-site_scripting). Only use `v-html` on trusted content and **never** on user-provided content.</p>
+  <p class="tip">Renderizar dinámicamente HTML arbitrario en su sitio web puede ser muy peligroso porque puede conducir fácilmente a [ataques XSS](https://en.wikipedia.org/wiki/Cross-site_scripting). Solo use `v-html` en contenido confiable y **nunca** en contenido provisto por el usuario.</p>
 
-  <p class="tip">In [single-file components](../guide/single-file-components.html), `scoped` styles will not apply to content inside `v-html`, because that HTML is not processed by Vue's template compiler. If you want to target `v-html` content with scoped CSS, you can instead use [CSS modules](https://vue-loader.vuejs.org/en/features/css-modules.html) or an additional, global `<style>` element with a manual scoping strategy such as BEM.</p>
+  <p class="tip">En [Componentes de un solo archivo](../guide/single-file-components.html), los estilos `scoped` no aplicarán al contenido dentro de `v-html`, porque el HTML no será procesado por el compilador de pantillas de Vue. Si usted quiere aplicar estilos _scoped_ al contendio `v-html`, puede utilizar [módulos CSS](https://vue-loader.vuejs.org/en/features/css-modules.html) o un elemento `<style>` global adicional, con una estrategia como BEM por ejemplo. </div>
 
-- **Example:**
+- **Ejemplo:**
 
   ```html
   <div v-html="html"></div>
   ```
 
-- **See also:** [Data Binding Syntax - Interpolations](../guide/syntax.html#Raw-HTML)
+- **Vea también:** [Sintaxis de Template - Interpolaciones](../guide/syntax.html#HTML-Puro)
```

Comment:
Shouldn't be something lik:

```suggestion
- **Vea también:** [Sintaxis de enlace de datos - Interpolaciones](../guide/syntax.html#Raw-HTML)
```

Reply:
Same here! Check: https://es.vuejs.org/v2/guide/syntax.html#HTML-Puro. Also, the name of that section is `Sintaxis de Template - Interpolaciones`, that's why I went with that translation 🤔